modules/log6x-input-spec-filter-namespace-container.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-input-spec-filter-namespace-container_{context}"]
+= Filtering application logs at input by including or excluding the namespace or container name
+
+You can include or exclude the application logs based on the namespace and container name by using the `input` selector.
+
+.Prerequisites
+
+* You have installed the {clo}.
```

Comment:
Also, added to these Prerequisites it should be recommended to have created the ClusterLogging CR and setting the limits for the collector as requested in documentation Bug - https://issues.redhat.com/browse/OBSDOCS-856. If this is explained in a different section of the Documentation, add the link to it